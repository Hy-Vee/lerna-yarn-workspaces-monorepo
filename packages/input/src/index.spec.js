import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Input from '.';

describe('Input', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Input />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
