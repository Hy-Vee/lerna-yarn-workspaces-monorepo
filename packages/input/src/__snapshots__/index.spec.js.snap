// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders correctly 1`] = `
.c0 {
  border: 1px solid #ccc;
  border-radius: 4px;
  box-sizing: border-box;
  font-size: 16px;
  font-weight: 300;
  padding: 10px 40px 10px 10px;
  width: 150px;
}

<input
  className="c0"
/>
`;
